src/main/java/refappintegrasjon/controller/Controller.java

```diff
@@ -0,0 +1,25 @@
+package refappintegrasjon.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import refappintegrasjon.consumer.EndpointProtectedByMaskinportenConsumer;
+
+@Slf4j
+@RestController
+public class Controller {
+
+    private EndpointProtectedByMaskinportenConsumer endpointProtectedByMaskinportenConsumer;
+
+    public Controller(EndpointProtectedByMaskinportenConsumer endpointProtectedByMaskinportenConsumer) {
+        this.endpointProtectedByMaskinportenConsumer = endpointProtectedByMaskinportenConsumer;
+    }
+
+    @RequestMapping(method = RequestMethod.GET,
+            value = "test/maskinportintegrasjon")
```

Comment:
småpirk, men trenger kanskje ikke denne linebreaken for lesbarhet :) og det vil kanskje være mer leselig å bruke @GetMapping("urlurl") istedenfor